src/pages/about-us/get-involved.tsx

```diff
@@ -109,59 +111,84 @@ const fixtureData: GetInvolvedPage["pageData"] = {
-        title={pageData.header.title}
-        content={pageData.header.text}
-        imageUrl={
-          "https://res.cloudinary.com/oak-web-application/image/upload/v1734018530/OWA/illustrations/auth-acorn_zyoma2.svg"
-        }
-        imageAlt={""}
-      />
-      <TodoSection
-        name="collab"
-        text={JSON.stringify(pageData.collab, null, 2)}
-      />
-      <TodoSection
-        name="workWithUs"
-        text={JSON.stringify(pageData.workWithUs, null, 2)}
-      />
-      <WhoAreWeExplore
-        title={"Explore more about Oak"}
-        items={[
-          {
-            iconName: "logo",
-            title: "About Oak",
-            href: "#",
-          },
-          {
-            iconName: "homepage-teacher-map",
-            title: "About Oak’s curriculum",
-            href: "#",
-          },
-          {
-            iconName: "data",
-            title: "Oaks impact",
-            href: "#",
-          },
-          {
-            iconName: "snack-break",
-            title: "Meet the team",
-            href: "#",
-          },
-        ]}
-      />
+      <OakBox $overflow={"hidden"}>
+        <WhoAreWeHeader
+          title={pageData.header.title}
+          content={pageData.header.text}
+          imageUrl={
+            "https://res.cloudinary.com/oak-web-application/image/upload/v1734018530/OWA/illustrations/auth-acorn_zyoma2.svg"
+          }
+          imageAlt={""}
+        />
+        <TodoSection
+          name="collab"
+          text={JSON.stringify(pageData.collab, null, 2)}
+        />
+        <GetInvolvedWorkWithUs
+          heading="Work with us"
+          text={[
+            "We're a fast-paced and innovative team, working to support and inspire teachers to deliver great teaching, so every pupil benefits.",
+            "All our roles are remote-first. If you want to be part of something unique that's making a difference to millions of children's lives, we'd love to hear from you.",
+          ]}
+          permanentRolesLink="https://app.beapplied.com/org/1574/oak-national-academy/"
+          freelanceRolesLink="https://app.beapplied.com/org/1767/oak-national-academy-freelancers/"
+          imageUrl="https://res.cloudinary.com/oak-web-application/image/upload/v1764066578/about-us/team-huddle_zivgxj.png"
+          imageAlt=""
+          badges={[
+            {
+              url: "https://res.cloudinary.com/oak-web-application/image/upload/v1764066553/about-us/top-1-percent-logo_hyga8g.svg",
+              alt: "Top 1% Employer badge",
+            },
+            {
+              url: "https://res.cloudinary.com/oak-web-application/image/upload/v1764066553/about-us/investor-in-people_eymeqv.svg",
+              alt: "Investors in People Gold badge",
+            },
+            {
+              url: "https://res.cloudinary.com/oak-web-application/image/upload/v1764066553/about-us/disability-confident_ym07wl.png",
+              alt: "Disability Confident Committed badge",
```

Comment:
The alt text for these badges is specified in the design docs (e.g. `‘In Escape the City's top 1% of employers’`) - can you correct this here? I'm aware that this will soon be in sanity anyway)